src/main/resources/configuration/VanillaCulturesConfiguration.json

```diff
@@ -74,23 +74,18 @@
   "yakut",
   "tibetan",
   "assamese",
-  "bengali",
   "bihari",
   "manipuri",
   "nepali",
   "oriya",
   "sinhala",
   "avadhi",
-  "kanauji",
```

Comment:
This culture is gone from culture files and all game config files except localization. Seems to have been removed.

Reply:
The other cultures just got moved around due to changing short positions in the game culture files.